tools/_release.sh

```diff
@@ -264,6 +264,14 @@ if [ "$RELEASE_BRANCH" = candidate-"$version" ] ; then
     SetVersion "$nextversion".dev0
     letsencrypt-auto-source/build.py
     git add letsencrypt-auto-source/letsencrypt-auto
+    for pkg_dir in $SUBPKGS_NO_CERTBOT certbot-compatibility-test .
+    do
+      sed -i 's/-e acme[dev]/acme[dev]==$version/' "$pkg_dir/local-oldest-requirements.txt"
```

Comment:
Can we check for the existence of this file first? It doesn't exist for each package and the absence of the file causes the release script to fail.

Reply:
why doesn't it exist for each package

Reply:
`acme` for example doesn't depend on any local packages in the repo.

Reply:
is it just acme that's excluded?

Reply:
I think just that and `certbot-compatibility-test`. We don't release `certbot-compatibility-test` so trying to make sure it works with new and old versions of our packages seems unnecessary to me as it's just a developer tool.